src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -10,41 +10,41 @@
 					<hero-background />
 				</div>
 				<content-container class="md:tw-pt-6 lg:tw-pt-8">
-					<summary-card class="tw-relative lg:tw--mb-1.5" />
+					<summary-card class="tw-relative lg:tw--mb-1.5 tw-z-1" />
```

Comment:
Application of these classes and the relative classes below did not change the layout from what I have seen. The relative classes adjusted the flow so that the content in those containing components was not obscured by the LendCTA component and it's full width content area.